packages/dds/tree/src/feature-libraries/index.ts

```diff
@@ -43,6 +43,7 @@ export {
 	ModularEditBuilder,
 	type FieldEditDescription as EditDescription,
 	type FieldChangeHandler,
+	type FieldChangeDelta,
```

Comment:
I'm hoping you won't need to export this anymore core/tree/deltaUtil.ts doesn't import it.
It's okay for tests to reach into modules to import stuff if needed.